Add eslint config

nicoledanuwidjaja opened the pull request:
Usage: `npm run lint`

Adds eslint checker to repository
Issue #32

avaer commented:
The eslint config makes sense, but can we hold off on re-linting the entire repo? It is already conflicting and there are probably going to be many other conflicts with things in flight if we do it now.

It should make more sense to re-lint after we split everything out into separate repos.

nicoledanuwidjaja commented:
> The eslint config makes sense, but can we hold off on re-linting the entire repo? It is already conflicting and there are probably going to be many other conflicts with things in flight if we do it now.
> 
> It should make more sense to re-lint after we split everything out into separate repos.

Updated to only include config and command line argument!

avaer commented:
Thanks!
